--- REV-plugins-0.7.1/Makefile.orig	2022-01-23 19:49:10.480719586 +0100
+++ REV-plugins-0.7.1/Makefile	2022-01-23 20:10:49.480348969 +0100
@@ -19,21 +19,22 @@
 # -----------------------------------------------------------------------
 
 
+CXX = g++
 CPPFLAGS += -I. -fPIC -D_REENTRANT
-CXXFLAGS += -Wall -O2
+CXXFLAGS += -Wall
 
 all:	zita-reverbs.so
 
 REVERBS_O = reverbs.o reverbs_if.o zreverb.o pareq.o
 zita-reverbs.so:	$(REVERBS_O)	
-	g++ -shared  -o $@  $(REVERBS_O)
+	$(CXX) $(LDFLAGS) $(CXXFLAGS) -shared  -o $@  $(REVERBS_O)
 
 reverbs.o:	ladspaplugin.h reverbs.h zreverb.h pareq.h
 reverbs_if.o:	ladspaplugin.h reverbs.h zreverb.h pareq.h
 
 
 install:	zita-reverbs.so
-	cp  *.so /usr/lib/ladspa
+	cp -p *.so $(DESTDIR)$(INSTALL_PLUGINS_DIR)
 
 clean:
 	rm -rf *~ *.o *.so
--- REV-plugins-0.7.1/ladspaplugin.h.orig	2022-01-23 19:49:10.480719586 +0100
+++ REV-plugins-0.7.1/ladspaplugin.h	2022-01-23 20:11:23.316832328 +0100
@@ -23,7 +23,7 @@
 #define __LADSPAPLUGIN_H
 
 
-#include "ladspa.h"
+#include <ladspa.h>
 
 
 class LadspaPlugin
