--- REV-plugins-0.8.1/source/Makefile.orig	2023-09-10 10:54:35.939012969 +0200
+++ REV-plugins-0.8.1/source/Makefile	2023-09-10 10:55:30.862048759 +0200
@@ -19,21 +19,22 @@
 # -----------------------------------------------------------------------
 
 
+CXX = g++
 CPPFLAGS += -I. -fPIC -D_REENTRANT
-CXXFLAGS += -Wall -O2
+CXXFLAGS += -Wall
 
 all:	zita-reverbs.so
 
 REVERBS_O = revplugins.o reverbs.o zreverb.o pareq.o
 zita-reverbs.so:	$(REVERBS_O)	
-	$(CXX) -shared  -o $@  $(REVERBS_O)
+	$(CXX) $(LDFLAGS) $(CXXFLAGS) -shared  -o $@  $(REVERBS_O)
 
 reverbs.o:	ladspaplugin.h reverbs.h zreverb.h pareq.h
 revplugins.o:	ladspaplugin.h reverbs.h zreverb.h pareq.h
 
 
 install:	zita-reverbs.so
-	cp  *.so /usr/lib/ladspa
+	cp -p *.so $(DESTDIR)$(INSTALL_PLUGINS_DIR)
 
 clean:
 	rm -rf *~ *.o *.so
--- REV-plugins-0.8.1/source/ladspaplugin.h.orig	2023-09-10 10:54:35.942346284 +0200
+++ REV-plugins-0.8.1/source/ladspaplugin.h	2023-09-10 10:56:00.868552866 +0200
@@ -22,7 +22,7 @@
 #define __LADSPAPLUGIN_H
 
 
-#include "ladspa.h"  // use local version
+#include <ladspa.h>
 
 
 class LadspaPlugin
